### ISSUE TITLE
Exploring: Who is using ngx-bootstrap?

### ISSUE THREAD

**Issue by ludmilanesvitiy** [valor-software/ng2-bootstrap#5168]
In the next few versions, we're planning to create a section called "**Who is using ngx-bootstrap**". We know that a lot of sites are using our library, and now it will be good to add a Companies List to the documentation.

If there are any companies that would like to be listed on our "Who is using ngx-bootstrap" page, just _post in the comment_ section below:

- Your company name

- Website URL

- Logo image

**Comment by livthomas** [comment]
We use `ngx-bootstrap` in our SaaS product called **Lumeer.io** which you can find on our website https://www.lumeer.io/. Feel free to use any of these logos:

- https://www.lumeer.io/wp-content/themes/lumeer/assets/img/logo-small.svg
- https://www.lumeer.io/wp-content/themes/lumeer/assets/img/logo-big.svg

**Comment by Lolmewn** [comment]
We use `ngx-bootstrap` in our SaaS products that give users insight in their power/gas consumption and other data that we collect from our clients.
Our website is https://sustainablebuildings.nl
Logo is attached below (it looks funny since a large portion of the image is white text on a transparent background)
![logo](https://user-images.githubusercontent.com/663563/57700709-ceca2800-765a-11e9-9484-a7a9bb1e6485.png)

**Comment by RakeshMaddala** [comment]
We use this framework for an internal tool at Dell Emc

**Comment by funregis** [comment]
- société générale
- intranet websites
- https://fr.wikipedia.org/wiki/Fichier:Soci%C3%A9t%C3%A9_G%C3%A9n%C3%A9rale.svg

**Comment by FrancescoBorzi** [comment]
We use ngx-bootstrap at **AzerothCore**, a no-profit organisation composed by volunteers who have a passion for open source software and gaming. 

Official website: http://www.azerothcore.org/
Official logo: http://www.azerothcore.org/images/logo.png
Linkedin page: https://www.linkedin.com/company/azerothcore/

**Comment by zaiddabaeen** [comment]
ngx-bootstrap is used heavily in our editor at **DeveloperHub.io**, it provides most dropdowns and modals.

Company Name: DeveloperHub.io
Website: https://developerhub.io
Logo image: https://res.cloudinary.com/developerhub/image/upload/v1561908888/1/gmoiyrndwsboeffgiz1x.svg

**Comment by cantacell** [comment]
we use ngx-bootstrap at trovimap.com

**Comment by Lolmewn** [comment]
> We use `ngx-bootstrap` in our SaaS products that give users insight in their power/gas consumption and other data that we collect from our clients.
> Our website is https://sustainablebuildings.nl
> Logo is attached below (it looks funny since a large portion of the image is white text on a transparent background)
> ![logo](https://user-images.githubusercontent.com/663563/57700709-ceca2800-765a-11e9-9484-a7a9bb1e6485.png)

I found a better logo (noticed the one on the discover page of your website looks real bad due to it being transparent with white text):
![sb](https://user-images.githubusercontent.com/663563/70622944-3eb68600-1c1d-11ea-82d3-7ccbd75befe8.png)

**Comment by hoclun-rigsep** [comment]
FDNY is using this in at least one application.

**Comment by daniloff200** [comment]
@Lolmewn  gonna update it ! Thanks!

@hoclun-rigsep  do you mean the Fire Department of NY? :fearful:

**Comment by hoclun-rigsep** [comment]
Yes.

**Comment by daniloff200** [comment]
@hoclun-rigsep  woah! :eyes: 
do you have some more info about that? I've just found https://www1.nyc.gov/site/fdny/codes/fire-code/fire-code.page,  and don't see any Angular stuff here.

**Comment by hoclun-rigsep** [comment]
Yeah, it's not public-facing stuff, but there's at least one large internal application using it.

**Comment by vvolodin** [comment]
Actually thinking about abandoning it. ngx-bootstrap clearly can't keep up neither with recent Angular changes nor its own growing list of issues...

**Comment by imtiazShakil** [comment]
We are using ngx-bootstrap for our client [AtMetis](http://atmetis.nl/) which is one of the top risk assessment company in the Netherlands.

It's an internal web-application which the company's engineer and staffs use.

- Your company name: AtMetis
- Website URL: https://www.atmetis.nl/
- Logo image: https://www.atmetis.nl/wp-content/uploads/2017/08/cropped-logo-atmetis-tagline-rgb1500px-1.png

**Comment by UdayAppam** [comment]
[Pramati](https://www.pramati.com/) is  using ngx-bootstrap for [wavemaker](https://www.wavemaker.com/) product.

WaveMaker, Inc. brings the WaveMaker Platform that combines rapid application Development (RAD) with PAAS (Platform As A Service) for customers wanting speedy creation of custom Apps that are browser delivered (responsive web) or device installed (hybrid mobile apps). WaveMaker removes bottlenecks in building Apps* with good UI, using popular frameworks and industry-standard products. WaveMaker Apps are integration ready (with RESTful API) and scale well on cloud deployment.

-  Company : **Pramati Technologies Private Limited.**
-  Company website: https://www.pramati.com/
- Company logo: https://www.pramati.com/wp-content/themes/twentynineteen-child/images/logo.svg
-  Product website:  https://www.wavemaker.com
-  Product logo: https://cdn.shortpixel.ai/spai/ret_img/https://www.wavemaker.com/wp-content/uploads/wavemaker-new-design-logo.svg

**Comment by delafer** [comment]
We are using ngx-bootstrap for our internal projects / clients.

1. Company: Creditreform e.V.
2. [Company website](https://www.creditreform.de)
3. [Company logo](https://de.m.wikipedia.org/wiki/Datei:Creditreform_logo.png)

**Comment by jaddap2** [comment]
I have been using ngx-boostrap until couple of weeks ago. I was only using modal and alert components. But during build I see all of the components are being used, so I replaced it with `Mat-Dialog` and custom `Alert` component. Removed ngx-bootstrap from my dependency list

**Comment by jurienhamaker** [comment]
**Your company name:** Employes
**Website URL:** https://app.employes.nl/
**Logo image:** https://app.employes.nl/assets/images/employes_logo_white.svg

**Comment by Jakartoine** [comment]
[EASI](https://easi.net/en) is a cloud, security & software provider offering management applications, professional cloud & security solutions and IT infrastructure services.

Two of our main products use ngx-bootstrap as the main UI external library:
- [SmartSales](https://easi.net/en/solutions/smartsales)
- [Rooftop](https://www.getrooftop.io/)

-----------------------------------------------------------------------------------------------------------------------------

- **Company name**: Easi
- **Website URL**: https://easi.net/en
- **Logo image**: https://easi.net/themes/custom/mia_interactive/images/easi-logo.svg

Please @ludmilanesvitiy, @daniloff200, consider adding our well-known Belgian company into your list.
We would all be grateful.

Thanks in advance,

Cheers

**Comment by b2k** [comment]
We use ngx-bootstrap for multiple SAAS product lines, kiosk and native electron mobile apps.

- **Company name:** Quotepro Inc
- **Website URL:** https://quoteprosolutions.com
- **Logo Image:** https://quoteprosolutions.com/wp-content/uploads/2019/05/QuotePro_Horizontal.png

**Comment by hoclun-rigsep** [comment]
Confirm FDNY, it's not a public facing app but it is in production with ~8k users, it's called "E–Staffing."
